code/Metalama.Documentation.SampleCode.Contracts/NotEmptyCollectionContract.t.cs

```diff
@@ -24,9 +24,9 @@ public Submenu([Required] string name, [NotNull][NotEmpty] IReadOnlyCollection<M
           throw new ArgumentOutOfRangeException("name", "The 'name' parameter is required.");
         }
       }
-      if (items.Count <= 0)
+      if (items == null !)
```

Comment:
This is wrong. A contract can't check for `items.Count <= 0` first and only then for `items == null`.

This probably requires changing the priority in Patterns.